package.json

```diff
@@ -7,11 +7,18 @@
   },
   "scripts": {
     "test": "mocha",
+    "test-with-coverage": "nyc mocha",
```

Comment:
@wcater803  please, can you add a small note to the README.md explaining how we should use this ? something simple that helps me get started. 

As of today we mention that to run these tests, after getting the services up and running with `podman-compose` you get the tests run with `npm test`. If would be neat to have a second line to say how to produce the coverage report, at least.

Thanks ! Other than that this looks good to me, awesome work !! :)